packages/components-stencil/src/components/card/readme.md

```diff
@@ -0,0 +1,21 @@
+# t-button
```

Comment:
Same here - shouldnt it be called card instead of t-button?

Reply:
fixed